ebean-core/src/main/java/io/ebeaninternal/api/LoadManyRequest.java

```diff
@@ -117,15 +106,7 @@ public SpiQuery<?> createQuery(SpiEbeanServer server) {
     }
     // potentially changes the joins, selected properties, cache mode
     loadContext.configureQuery(query);
-    if (onlyIds) {
-      // lazy loading invoked via clear() and removeAll()
-      String mapKey = many.mapKey();
-      if (mapKey != null) {
-        query.select(mapKey);
-      } else {
-        query.select(many.targetIdProperty());
-      }
-    }
```

Comment:
CHECKME: Is ist ok, if the `onlyIds` code path is in `newLoadManyQuery` or may there be cases, where configureQuery overwrites the select?